src/caldav/bin/patch-sabredav-max-date.php

```
@@ -0,0 +1,55 @@
+
+$target = __DIR__ . '/../vendor/sabre/dav/lib/CalDAV/Backend/PDO.php';
+if (!file_exists($target)) {
+    fwrite(STDERR, "patch-sabredav-max-date: file not found: $target\n");
+    exit(1);
+}
+
+$needle  = "const MAX_DATE = '2038-01-01';";
+$replace = "const MAX_DATE = '2200-01-01';";
+$source  = file_get_contents($target);
+
+$count = substr_count($source, $needle);
```

PHPStan (2.1.55)

[warning] Parameter `#1` $haystack of function substr_count expects string, string|false given.


(argument.type)